src/main/g8/README.md

```diff
@@ -72,6 +72,28 @@ sbt -Ddocker.username=mary docker:publish
 Refer to [Configuring registries](https://developer.lightbend.com/docs/akka-serverless/projects/container-registries.html)
 for more information on how to make your Docker image available to Akka Serverless.
 
-Finally you can or use the [Akka Serverless Console](https://console.akkaserverless.com)
+Finally you can use the [Akka Serverless Console](https://console.akkaserverless.com)
 to create a project and then deploy your service into the project
-through the `akkasls` CLI.
+through the [akkasls](https://developer.lightbend.com/docs/akka-serverless/akkasls/using-cli.html) CLI:
```

Comment:
It's actually no longer possible to use Akka Serverless Console to create a project. You can only use it to view projects.

Instead I would say something like this:

`Use the [akkasls](https://developer.lightbend.com/docs/akka-serverless/akkasls/using-cli.html) CLI to create a project and deploy your service into it.`

Reply:
Ah, you cannot create a new project, but you will create an initial project in the welcome wizard.

So perhaps here we should just show how to deploy a service in that project?

![new-project-wizard](https://user-images.githubusercontent.com/131856/151511556-0c36ec70-0b71-491f-8863-95fd0f2af36f.png)